plugins/modules/sonic_route_maps.py

```diff
@@ -0,0 +1,1608 @@
+                  - none
+                  -   Do not send any community attribute. This attribute
+                  -   is mutually exclusive with all other 'set community'
+                  -   attributes. It causes all attributes to be removed
+                  -   rom the matching route.
+                type: list
+                elements: str
+                choices:
+                  - local_as
+                  - no_advertise
+                  - no_export
+                  - no_peer
+                  - additive
+                  - none
+          extcommunity:
+            description:
+              - BGP extended community attributes to set into a matching route.
+            type: dict
+            suboptions:
+              rt:
+                description:
+                  - Route Target VPN extended communities in the format
+                  - "ASN:NN or IP-ADDRESS:NN"
+                  - "Note: Each rt value in the list must be enclosed"
+                  - in double quotes to avoid YAML parsing errors due to the
+                  - "list values contaning an embedded ':' character."
+                type: list
+                elements: str
+              soo:
+                description:
+                  - "Site-of-Origin VPN extended communities in the format"
+                  - "ASN:NN or IP-ADDRESS:NN"
+                  - "Note: Each rt value in the list must be enclosed"
+                  - in double quotes to avoid YAML parsing errors due to the
+                  - "list values contaning an embedded ':' character."
+                type: list
+                elements: str
+          ip_next_hop:
+            description:
+              - IPv4 next hop address to set into a matching route in the
+              - dotted decimal format A.B.C.D
+            type: str
+          ipv6_next_hop:
+            description:
+              - IPv6 next hop address attributes to set into a matching route
+            type: dict
+            suboptions:
+              global_addr:
+                description:
+                  - IPv6 global next hop address to set into a matching
+                  - "route in the format A::B"
+                type: str
+              prefer_global:
+                description:
+                  - Set the corresponding attribute into a matching route
+                  - if the value of this Ansible attribute is 'true'.
+                  - The attribute indicates that the routing algorithm must
+                  - prefer the global next-hop address over the link-local
+                  - address if both exist.
+                type: bool
+          local_preference:
+            description:
+                - "BGP local preference path attribute; integer value in"
+                - the range 0-4294967295
+            type: int
+          metric:
+            description:
+              - route metric value actions
```

Comment:
Please specify that the metric options are mutually exclusive.

Reply:
Done